src/lints/pub_static_mut_now_immutable.ron

```diff
@@ -0,0 +1,50 @@
+SemverQuery(
+    id: "pub_static_mut_now_immutable",
+    human_readable_name: "pub static mut is now immutable",
+    description: "A mutable static became immutable and thus cannot be imported",
+    required_update: Major,
+    reference_link: Some("https://doc.rust-lang.org/reference/items/static-items.html"),
+    query: r#"
+    {
+        CrateDiff {
+            baseline {
+                item {
+                    ... on Static {
+                        visibility_limit @filter(op: "=", value: ["$public"])
+                        mutable @filter(op: "=", value: ["$true"])
+
+                        importable_path {
+                            path @output @tag
+                            public_api @filter(op: "=", value: ["$true"])
+                        }
+                    }
+                }
+            }
+            current {
+                item {
+                    ... on Static {
+                        visibility_limit @filter(op: "=", value: ["$public"])
+                        mutable @filter(op: "!=", value: ["$true"])
+                        static_name: name @output
+
+                        importable_path {
+                            path @filter(op: "=", value: ["%path"])
+                            public_api @filter(op: "=", value: ["$true"])
+                        }
+
+                        span_: span @optional {
+                            filename @output
+                            begin_line @output
+                        }
+                    }
+                }
+            }
+        }
+    }"#,
+    arguments: {
+        "public": "public",
+        "true": true,
+    },
+    error_message: "A mutable static is now immutable",
```

Comment:
We may want to expand this error message a bit.

If you just ran `cargo-semver-checks` and it reported this problem, what information would you like to see printed as part of the error log? You'd like to know what happened, why it's a problem, and where to look for it -- in as few characters as possible. This is tough, but it's a skill like any other and practicing it will help.